packages/app/script-utils/vite.config.ts

```diff
@@ -0,0 +1,11 @@
+import { resolve } from 'path'
```

Comment:
```suggestion
import { resolve } from 'node:path'
```

Reply:
Done. It was copied from other packages. Let me change it everywhere.

Reply:
Changed in other places too :blush: